src/main/java/apoc/gephi/GephiFormatUtils.java

```diff
@@ -0,0 +1,18 @@
+package apoc.gephi;
+
+import apoc.export.util.FormatUtils;
+import org.neo4j.graphdb.Node;
+
+public class GephiFormatUtils {
+
+    public static String captionGraphml(Node n, String[] captions) {
+        for (String caption : captions) {
+            if (n.hasProperty(caption)) return n.getProperty(caption).toString();
```

Comment:
first do one loop with the exact names 
2nd loop with lowercase
3rd loop with contains or endsWith